Jenkinsfile

```diff
@@ -31,14 +31,13 @@
  *     - Extended: 4 workers, 300 seconds
  *     - Stress: 100 workers, 1800 seconds
  * - Misc. Tasks:
- *     - Build Platinum AMI (optional)
+ *
  */
```

Comment:
I think it would be fine to remove the entire "Optionality" comment section.

Reply:
Done